app/src/main/res/values-nl/strings.xml

```diff
@@ -0,0 +1,259 @@
+    <string name="color_teal">Teal</string>
+    <string name="theme_dark">Donker</string>
+    <string name="theme_light">Licht</string>
+    <string name="theme_auto">Automatisch</string>
+    <string name="playlist_maximum_prefix">Sta niet meer toe dan</string>
+    <string name="playlist_maximum_suffix">nummers</string>
+    <string name="playlist_chosen_by_prefix">gekozen door</string>
+    <string name="match_all_rules">Voldoe aan alle regels</string>
+    <string name="rule_is">Is</string>
+    <string name="rule_isnt">Is niet</string>
+    <string name="rule_name_is">Naam is</string>
+    <string name="rule_name_isnt">Naam is niet</string>
+    <string name="rule_name_contains">Naam bevat</string>
+    <string name="rule_name_doesnt_contain">Naam bevat geen</string>
+    <string name="rule_play_count_lt">Is minder afgespeeld dan</string>
+    <string name="rule_play_count_eq">Is net zo vaak afgespeeld als</string>
+    <string name="rule_play_count_gt">Is vaker afgespeeld dan</string>
+    <string name="rule_skip_count_lt">Is minder vaak overgeslagen dan</string>
+    <string name="rule_skip_count_gt">Is vaker overgeslagen dan</string>
+    <string name="rule_skip_count_eq">Is net zo vaak overgeslagen als</string>
+    <string name="rule_added_before">Was toegevoegd voor</string>
+    <string name="rule_added_on">Was toegevoegd op</string>
+    <string name="rule_added_after">Was toegevoegd na</string>
+    <string name="rule_played_before">Was laatst gespeeld voor</string>
+    <string name="rule_played_on">Was laatst gespeeld op</string>
+    <string name="rule_played_after">Is gespeeld sinds</string>
+    <string name="enable_multi_repeat_title">Multi-herhalen inschakelen</string>
+    <string name="enable_sleep_timer">Slaaptimer</string>
+    <string name="confirm_sleep_timer_end">De slaaptimer is geëindigd</string>
+    <string name="empty_excluded_dirs">Er worden geen mappen uitgesloten</string>
+    <string name="empty_queue_detail">Je kan muziek afspelen vanuit de bibliotheek</string>
+    <string name="empty_recently_added_detail">Muziek die je in de laatste 30 dagen hebt toegevoegd zal hier verschijnen</string>
+    <string name="empty_included_dirs">Alle mappen worden ingesloten</string>
+    <string name="about_version">Versie</string>
+    <string name="empty_directory">Deze map heeft geen mappen</string>
+    <string name="about_open_privacy_policy">Privacybeleid</string>
+    <string name="lastfm_header">Artiesteninformatie wordt aangeboden door</string>
+    <string name="button_play_description">Afspeelknop</string>
+    <string name="button_skip_description">Volgende nummer knop</string>
+    <string name="button_previous_description">Vorige nummer knop</string>
+    <string name="description_album_art">Album Art</string>
+    <string name="player_notification_channel_name">Muziek Afspelen</string>
+    <string name="hint_playlist_name">Afspeellijst naam</string>
+    <string name="time_in_min_format">%1$d min</string>
```

Comment:
"Min" here is short for "Minutes," which translates to "Notulen" according to Google Translate. Do you know what a similar abbreviation would be for this?

Reply:
Again, Google Translate is wrong here...

"Notulen" means "notes" in English (as in, taking notes in class)